lib/vector-common/src/request_metadata.rs

```diff
@@ -1,44 +1,18 @@
+use std::collections::HashMap;
 use std::ops::Add;
-use std::{collections::HashMap, sync::Arc};
 
 use crate::{
-    config::ComponentKey,
     internal_event::{
-        CountByteSize, InternalEventHandle, OptionalTag, RegisterTaggedInternalEvent,
-        RegisteredEventCache,
+        CountByteSize, InternalEventHandle, RegisterTaggedInternalEvent, RegisteredEventCache,
+        TaggedEventsSent,
     },
     json_size::JsonSize,
 };
 
-/// Tags that are used to group the events within a batch for emitting telemetry.
-#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct EventCountTags {
-    pub source: OptionalTag<Arc<ComponentKey>>,
-    pub service: OptionalTag<String>,
-}
```

Comment:
Did you consider making this a type alias instead? i.e. `pub type EventCountTags = TaggedEventsSent;`

Reply:
I did and did have that type alias until just before pushing the PR. I'm not sure I like type aliases that just rename the type, since it's just an extra level of indirection to work out what is going on.

Happy to put it back in if you feel strongly about it.